python/pyspark/sql/tests/connect/test_connect_column_expressions.py

```diff
@@ -134,6 +138,37 @@ def test_list_to_literal(self):
         lit_list_plan = fun.lit([fun.lit(10), fun.lit("str")]).to_plan(None)
         self.assertIsNotNone(lit_list_plan)
 
+    def test_tuple_to_literal(self):
```

Comment:
QQ is this Python specific?

Reply:
yes, scala `lit` do not support tuple and map.